src/classes/CON_Phone.cls

```diff
@@ -314,11 +314,28 @@ public with sharing class CON_Phone {
                 'Mobile Phone' => 'Mobile',
                 'Other Phone' => 'Other'
             };
-
+            
             this.value = val;
             this.label = lbl;
             this.apiName = api;
-            this.prefLabel = (preferredPhoneMappings.containsKey(lbl)) ? preferredPhoneMappings.get(lbl) : lbl;
+            
+            String namespace = UTIL_Namespace.getNamespace();
+            String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
```

Comment:
Same question as above regarding prefix.

Reply:
That functionality would work if the label and picklist have an exact match.